vaadin-checkbox-flow-demo/pom.xml

```diff
@@ -7,7 +7,7 @@
     <parent>
         <groupId>com.vaadin</groupId>
         <artifactId>vaadin-checkbox-flow-parent</artifactId>
-        <version>2.0-SNAPSHOT</version>
```

Comment:
Nit: `notification` and `button` and `ordered-layout` snapshots versions can be bumped after other prs are merged.

Reply:
My intention is to do this round of PRs and only bumping these dependencies where they cause builds to fail, because there are so many of these that can be hard to find the right order and there might be some circular dependencies as well.
Then, in a second round, after all snapshots are available, I would bump these dependencies. WDYT?